src/app/pages/TransactionDetailPage/index.tsx

```diff
@@ -64,6 +65,19 @@ const StyledAlert = styled(Alert)(() => ({
   marginBottom: '1em',
 }))
 
+const ErrorBox = styled(Box)(() => ({
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  minHeight: '28px',
+  fontSize: '15px',
```

Comment:
wrong size ?

Reply:
In this case can you double check with Don, please?
<img width="879" alt="Screenshot 2023-03-17 at 09 21 34" src="https://user-images.githubusercontent.com/891392/225851193-a467b9b7-6612-4937-a248-a2f2aa39f4ad.png">

Reply:
Interesting, I definitely see 15px.

![image](https://user-images.githubusercontent.com/2093792/225853652-023d6679-ffd4-4c3a-a557-ee6a78f47e95.png)

Reply:
Waiting for the 15px vs 12px decision from @donouwens ...

Reply:
12px it is!